### DISCUSSION TITLE
Desktop Lockscreen

### DISCUSSION THREAD

**Discussion by Eltik** [rocksdanister/lively#1210] [Q&A] [answered]
Hello! I noticed that in an older version of Lively (https://www.youtube.com/watch?v=kWkHc_Qhts4) desktop lockscreens could display an animated wallpaper. However, in the System tab for the latest version, this option doesn't exist. Is this just me, or does the newest version not support the feature?

![image](https://user-images.githubusercontent.com/76538547/167238885-0fe1945b-abe2-42ed-89ea-4fde91c985a6.png)

OS: Windows 11 Home
Lively version: Latest (v1.7.5.2 I believe. I downloaded it from the Windows store)

**Comment by rocksdanister** [comment, marked as answer]
It's just a picture of the animated wallpaper as lockscreen, currently removed due to issue https://github.com/rocksdanister/lively/issues/537

**Reply by Eltik** [reply]
Ah okay. Thank you!
